src/platform/pret_sdl/sdl2.c

```diff
@@ -128,7 +128,7 @@ int main(int argc, char **argv)
 
     ReadSaveFile("sa2.sav");
 
-    if (SDL_Init(SDL_INIT_VIDEO | SDL_INIT_AUDIO) < 0) {
+    if (SDL_Init(SDL_INIT_VIDEO | SDL_INIT_AUDIO | SDL_INIT_JOYSTICK) < 0) {
```

Comment:
Read some threads somewhere that they saw better performance with this turned on